SS_readcontrol_330.tpl

```diff
@@ -4604,14 +4604,22 @@
           timevary_parm_cnt_sel++;
           N_selparm2++;
           ParmLabel += "sigmasel_" + fleetname(f) + "(" + NumLbl(f) + ")_" + anystring + "(" + NumLbl(max(1, j)) + ")";
+          if (dtempvec(7) > 0)
+          {
+            warnstream << "2DAR Sigmasel parameter is not normally estimable; adequate performance usually obtained by fixing to value of 1.0";
+            write_message (WARN, 0);
+          }
         }
         if (use_rho == 1)
         {
-          echoinput << "read two parameter lines for rho_yr and then rho_age (or length)" << endl;
+          warnstream << "2DAR rho is incompletely implemented; it should only be used experimentally and never estimated";
```

Comment:
I am hesitant to allow users to even do this if it is incompletely implemented. I would prefer a full stop if `use_rho = 1`. Especially because in this same if statement you set `dtempvec(7) = -1`. Which, does that just change the values put in ss_new or does it actually turn off estimation without the user knowing?

Reply:
The estimation has never been enabled.  The value of rho is only used in the preliminary calcs section to create a cor matrix.